packages/search-input/src/SearchResult/SearchResult.tsx

```diff
@@ -40,9 +40,10 @@ export const SearchResult = InferredPolymorphic<SearchResultProps, 'li'>(
       rest['aria-label'] ?? (rest['aria-labelledby'] ? '' : textContent);
 
     return (
+      // @ts-expect-error
```

Comment:
why?

Reply:
```
Types of property 'href' are incompatible.
        Type 'NodeUrlLike | ((string | NodeUrlLike) & string) | undefined' is not assignable to type 'string | NodeUrlLike'.
          Type 'undefined' is not assignable to type 'string | NodeUrlLike'.ts(2322)
```

Reply:
This exists even if we don't destructure as/href